src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp

```diff
@@ -195,5 +197,89 @@ void damped_harmonic(
     const tnsr::I<DataVector, SpatialDim, Frame>& coords, double amp_coef_L1,
     double amp_coef_L2, double amp_coef_S, int exp_L1, int exp_L2, int exp_S,
     double sigma_r);
+
+/*!
+ * \brief Impose damped harmonic gauge.
+ *
+ * \see `damped_harmonic()`
+ */
+class DampedHarmonic final : public GaugeCondition {
+ public:
+  /// The width of the Gaussian for the spatial decay of the damped harmonic
+  /// gauge.
+  struct SpatialDecayWidth {
+    using type = double;
+    static constexpr Options::String help{
+        "Spatial width of weight function used in the damped harmonic "
+        "gauge."};
```

Comment:
Since you use the names of the amplitudes $A_X$ and exponents $e_X$ below, maybe change this to
`Spatial width (sigma_r) of weight function (W) used ...` or something similar, just so symbols and names match up with what is in the equations.